doc/source/migration_details.rst

```diff
@@ -5,47 +5,62 @@ Check out the code manually and upgrade your database by calling odoo-bin,
-   integrity of your database may me lacking. Check the module coverage in
-   this documentation.
+Make the `openupgrade_framework` and the `openupgrade_scripts` modules
+available in the addons path in the Odoo instance of the new version.
 
-3. Decide which database you are going to upgrade. You absolutely *must* make a
-   backup of your live database before you start this process!
+Or, for older versions: check out the OpenUpgrade source code from Github
+for the branches you need. Each branch migrates to its version from the
+previous version, so branch 13.0 migrates from 12.0 to 13.0. If you are
+migrating across multiple versions, you need to run each version of
+OpenUpgrade in order. Skipping versions is not supported.
 
-4. Edit the configuration files and command line parameters to point to the
-   database you are going to upgrade. Run Odoo with the *--update all
-   --stop-after-init --load=web,openupgrade_framework* flags.
+The OpenUpgrade repository includes both `openupgrade_framework` and
+`openupgrade_scripts`:
 
-   For versions earlier than 14.0 that are running the OpenUpgrade fork rather
-   than Odoo itself, you do not pass the `load` parameter.
+https://github.com/OCA/openupgrade
 
-5. Run the upgrade and check for errors. You will probably learn a lot about
-   your data and have to do some manual clean up before and after the upgrade.
-   Expect to repeat the process several times as you encounter errors, clean up
-   your data, and try again. If necessary, ask for help or report bugs on
-   Github.
-
-General Tips
-++++++++++++
+openupgradelib
+..............
 
 * When installing the openupgradelib make sure you check out the latest version
   from github to get the latest updates and fixes::
 
     pip install git+git://github.com/OCA/openupgradelib.git
 
+2. Check coverage of the migration scripts for your installed modules
+*********************************************************************
+
+Check if there are migration scripts provided for the set of modules that
+are installed in your Odoo database. If there are modules for which no
+migration scripts have been developed yet, your migration may fail or the
+integrity of your database may be lacking. Check the module coverage in
+this documentation under :doc:`status` and refer to the :doc:`development`
+documentation to add the missing migration scripts.
+
+3. Make a copy of the database to migrate
+*****************************************
+
+Decide which database you are going to upgrade. You absolutely *must* make a
+backup of your live database before you start this process!
+
+4. Adjust the configuration for Odoo and OpenUpgrade
+****************************************************
+
+Edit the configuration files and command line parameters to point to the
+database you are going to upgrade. The recommended command line parameters are
+the ```--update all --stop-after-init --load=web,openupgrade_framework`` flags.
```

Comment:
```suggestion
the ```--update all --stop-after-init --load=base,web,openupgrade_framework`` flags.
```

Reply:
Is that really necessary?
If it is, then we probably should also update this README as well:

https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_framework/README.rst

Reply:
Doing that here https://github.com/OCA/OpenUpgrade/pull/2629